noir-projects/noir-protocol-circuits/crates/rollup-lib/src/components.nr

```diff
@@ -225,6 +225,7 @@ fn get_tx_effects_hash_input(
 
     let mut offset = 0;
     let mut array_len = 0;
+    let mut check_elt = true;
```

Comment:
Nit but it's probably a good idea to avoid declaring mutable variables up at the top of functions like this when it's being used in multiple separate instances.

This makes it easier for us to verify that it's doing what we expect and also easier for the compiler to determine that we don't need to consider previous usage of this variable.

Reply:
Good point, done!